resources/views/Item/SingleItemWrapper.twig

```diff
@@ -102,6 +102,7 @@
                         item-data="{{ item | json_data }}"
                         attributes-data="{{ attributes | json_data }}"
                         variations="{{ variations | json_data }}"
+                        add-please-select-option-variation-id="{{ addPleaseSelectOptionVariationId | json_data }}"
```

Comment:
```suggestion
                        add-please-select-option-variation-id="{{ addPleaseSelectOptionVariationId | json_encode }}"
```

`json_data` sorgt dafür, dass der Wert in ein separates script-Tag geschrieben wird und die ID als referenz an die Komponente gegeben wird. Das ist sinnvoll bei sehr großen objekten, um sich die vielen `&quot` bei json-formatierten Objekten zu sparen. In dem Fall ist es overhead, da `addPleaseSelectOptionVariationId` nur ein Integer enthält.